### DISCUSSION TITLE
replaceInFile

### DISCUSSION THREAD

**Discussion by carstenjaksch** [laravel/framework#37119] [Ideas]
Hey,

I've spotted the `replaceInFile` function in [laravel/jetstream](https://github.com/laravel/jetstream/blob/37a0e52eb4cfb5408d22e67be73f9c32c084cdfa/src/Console/InstallCommand.php#L588) and wonder why this is not part of the official Laravel API?

The function could be used in many scaffolding packages. Even Laravel needs it [multiple times](https://github.com/laravel/breeze/blob/fab1ce25c0a181113eb9de9dfb2a2bfda1ee9bb2/src/Console/InstallCommand.php#L280). Not really DRY. 😉

**Comment by crynobone** [comment]
Already possible: https://github.com/laravel/framework/pull/38069
